recipes/skrebate/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "skrebate" %}
+{% set version = "0.4" %}
+{% set sha256 = "e7311e2ef54461a2d6df4ac4c81d74dec9e71ccd02040bc7fadb3a5221225cc2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - numpy
+    - python
+    - scikit-learn
+    - scipy
+
+test:
+  imports:
+    - skrebate
+
+about:
+  home: https://epistasislab.github.io/scikit-rebate
+  license: MIT
+  license_family: MIT
+  # TODO: upstream a PR to add LICENSE to MANIFEST.in
+  # license_file: LICENSE
```

Comment:
Please submit the PR and list it as a comment in this recipe. Also:

conda-forge.github.io/docs/meta.html#packaging-the-license-manually

Reply:
[PR](https://github.com/EpistasisLab/scikit-rebate/pull/31)!